hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/services/CompareIndexService.kt

```diff
@@ -37,7 +38,7 @@ class CompareIndexService(
 
   fun doIndexSizeCheck(): SizeCheck {
     val start = System.currentTimeMillis()
-    val totalIndexNumber = prisonerRepository.count(indexStatusService.getIndexStatus().currentIndex)
+    val totalIndexNumber = prisonerRepository.count(indexStatusService.getIndexStatus(INDEX_STATUS_ID).currentIndex)
```

Comment:
In order to generalise various index service classes the index id now needs to be specified